kyuubi-server/web-ui/tsconfig.json

```diff
@@ -0,0 +1,19 @@
+{
```

Comment:
does this file allow to add header as well? I know there are some json library extend the JSON standard to support comment syntax.

Reply:
Oh, really? could you show me how to add? As I researched and find nothing, also checked other repos, but their json also has no comments...

Reply:
https://github.com/apache/skywalking-booster-ui/blob/main/tsconfig.json